src/PAModel/Microsoft.PowerPlatform.Formulas.Tools.csproj

```diff
@@ -1,6 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
consider adding `net8.0` as a target (change to plural: `TargetFrameworks` even when there's only one).
this is good practice for allowing support of older frameworks, but to jump forward to latest .net when consumers support it.

Reply:
it means that doc svr will be able to utilize the net8.0 version of the dll

Reply:
Good idea.  Done.